apps/src/templates/SchoolAutocompleteDropdown.jsx

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react';
+import VirtualizedSelect from 'react-virtualized-select';
+import 'react-virtualized/styles.css';
+import 'react-select/dist/react-select.css';
+import 'react-virtualized-select/styles.css';
+
+export default class SchoolAutocompleteDropdown extends Component {
+
+  state = {};
+
+  render() {
+    const getOptions = (q) => {
```

Comment:
:+1: In fact, it looks like it could be defined outside of the component entirely, and doesn't need to be an arrow function, since it has no dependency on `this`.

Reply:
We should unit-test this function, either the whole thing using something like [Sinon's `FakeServer`](http://sinonjs.org/releases/v3.3.0/fake-xhr-and-server/), and/or extract the fairly complex logic from your second `then()` and test it in isolation.